From 11d2cb46509ef571db25d92f2093db783ae2f672 Mon Sep 17 00:00:00 2001
From: Stephen George <stephen.george@freescale.com>
Date: Wed, 14 Mar 2012 03:23:48 +0000
Subject: [PATCH 0522/1089] perf: Adding exports for perf functions for
 external PMU

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

Signed-off-by: Stephen George <stephen.george@freescale.com>
---
 kernel/events/core.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kernel/events/core.c b/kernel/events/core.c
index f774e93..0f6d6db 100644
--- a/kernel/events/core.c
+++ b/kernel/events/core.c
@@ -3809,6 +3809,7 @@ void perf_event_update_userpage(struct perf_event *event)
 unlock:
 	rcu_read_unlock();
 }
+EXPORT_SYMBOL_GPL(perf_event_update_userpage);
 
 static int perf_mmap_fault(struct vm_area_struct *vma, struct vm_fault *vmf)
 {
@@ -5405,6 +5406,7 @@ int perf_event_overflow(struct perf_event *event,
 {
 	return __perf_event_overflow(event, 1, data, regs);
 }
+EXPORT_SYMBOL_GPL(perf_event_overflow);
 
 /*
  * Generic software event infrastructure
-- 
2.0.2

